marketplaceItems/audioFocus/appResources/appData/resources/js/audioFocus_ui.js

```diff
@@ -0,0 +1,118 @@
+/*
+    Solo Point
+    Created by Milad Nazeri on 2019-01-07
+    Copyright 2019 High Fidelity, Inc.
+
+    Distributed under the Apache License, Version 2.0.
+    See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+    Point to solo someone to hear them better in a crowd!
+    TABLET JS
+    
+*/
+
+
+// *************************************
+// START UTILITY FUNCTIONS
+// *************************************
+// #region Utilty
+
+var debug = true;
+var FALSE = "false";
```

Comment:
i know we talked about this block of code in a previous review - leaving this message here so i have a comment associated with the latest commit